modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/reducers/pageValidationReducer.es.js

```diff
@@ -63,7 +65,14 @@ export default (state, action) => {
 			);
 
 			if (firstInvalidFieldInput) {
-				firstInvalidFieldInput.focus();
+				if (firstInvalidFieldInput.type !== 'hidden') {
+					firstInvalidFieldInput.focus();
+				}
+				else {
+					document
+						.getElementsByName(firstInvalidFieldName)[0]
+						.parentElement.scrollIntoView();
```

Comment:
@joaovictoralvesmartins, do you think there is any chance of we get an `undefined` element through the calls `getElementsByName(firstInvalidFieldName)[0]` and `parentElement`?

if so, then would be nice the check the existence of these elements before run `scrollIntoView`, what do you think?

Reply:
@natocesarrego, you're right! I modified the code, so it is checking now for the existence of the parent elements.